ui/src/app/views/workspace/component.spec.ts

```diff
@@ -121,6 +122,7 @@ describe('WorkspaceComponent', () => {
       });
     currentWorkspaceStore.next(workspaceDataStub);
     registerApiClient(CohortsApi, new CohortsApiStub());
+    registerApiClient(ConceptSetsApi, new ConceptSetsApiStub());
```

Comment:
Hmm, I don't see any relationship between this and the component being tested - were you seeing an error originate from here? If so what was it?

Reply:
It wasn't causing a test failure but it did throw a console error that looked very similar to the circle-ci flake so I thought it might help with flakes:

`./src/app/views/recent-work/component.tsx?:62 Error: API client is not registered: ensure you are not retrieving an API client before app initialization. In unit tests, be sure to call registerApiClient() for all API clients in use, else call bindApiClients(): function ConceptSetsApi() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
    at Object.eval [as conceptSetsApi] (./src/app/services/swagger-fetch-clients.ts?:48)
    at class_1.eval (./src/app/views/recent-work/component.tsx?:45)
    at step (./node_modules/tslib/tslib.es6.js?:117)
    at Object.eval [as next] (./node_modules/tslib/tslib.es6.js?:98)
    at eval (./node_modules/tslib/tslib.es6.js?:91)
    at new ZoneAwarePromise (./node_modules/zone.js/dist/zone.js?:890)
    at Module.__awaiter (./node_modules/tslib/tslib.es6.js?:87)
    at eval (./src/app/views/recent-work/component.tsx?:35)
    at class_1.eval (./src/app/views/recent-work/component.tsx?:55)
    at step (./node_modules/tslib/tslib.es6.js?:117)`

Reply:
Ah good catch - I see the dep now coming in from app-recent-work.